Cue/app/common/CueColors.js

```diff
@@ -5,6 +5,7 @@
 module.exports = {
   primaryAccent: '#51A7F9',
 
+  primaryTintEvenLighter: '#BED1EA',
```

Comment:
Actually this colour is darker than `primaryTintLighter` (which is `primaryTint` at 15% opacity and is the background colour for the user name in the drawer menu). 😬